functional-test/src/test/groovy/org/rundeck/tests/functional/schedule/ScheduleSpec.groovy

```diff
@@ -0,0 +1,115 @@
+package org.rundeck.tests.functional.schedule
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.rundeck.tests.functional.api.ResponseModels.CreateJobResponse
+import org.rundeck.tests.functional.api.ResponseModels.JobsExportResponse
+import org.rundeck.util.annotations.APITest
+import org.rundeck.util.api.JobUtils
+import org.rundeck.util.container.BaseContainer
+
+@APITest
+class ScheduleSpec extends BaseContainer{
+
+    def setupSpec() {
+        startEnvironment()
+        setupProject()
```

Comment:
This method is used to create the 'test' project, but I see that in the test you don't use that project.